app/controllers/task_controller.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+# Task controller
+class TaskController < ApplicationController
+  layout false
+  def index; end
+
+  def tasks_overview; end
```

Comment:
If the controller is `tasks` then the action could be named `overview` only, it's self-explanatory